### PR TITLE
Zed: Update to Rocky 9.3 based kolla images

### PR DESCRIPTION
**Pull request by jovial**
These have been rebuilt using Rocky 9.3 snapshots.

Builds:

neutron: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/7757168800
others: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/7754948585

### CODE REVIEW

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -12,22 +12,17 @@ kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %
...
-    rocky: zed-rocky-9-20230925T132313
     ubuntu: zed-ubuntu-jammy-20230821T155947
   cloudkitty:
-    rocky: zed-rocky-9-20231114T124701
     ubuntu: zed-ubuntu-jammy-20231114T124701
   neutron:
-    rocky: zed-rocky-9-20231115T094053
     ubuntu: zed-ubuntu-jammy-20231115T094053
   opensearch:
-    rocky: zed-rocky-9-20231214T095452
```
Comment (Alex-Welsh) [inline etc/kayobe/kolla/globals.yml]: Removing these lines will cause the tag lookup (lines 34-38) to break. I think in the past we've resolved this by either just duplicating the tag, or adding a default value to the lookup

File: etc/kayobe/kolla/globals.yml [hunk 2]
```diff
@@ -12,22 +12,17 @@ kolla_base_distro: "{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %
...
     ubuntu: zed-ubuntu-jammy-20231214T095452
 
 openstack_tag: "{% raw %}{{ kayobe_image_tags['openstack'][kolla_base_distro] }}{% endraw %}"
```
Comment (jovial) [inline etc/kayobe/kolla/globals.yml]: Hmm, no default on these now :(

File: etc/kayobe/kolla/globals.yml (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml]: If only we had merged https://github.com/stackhpc/stackhpc-kayobe-config/pull/663 already we wouldn't need to worry about this :D

Comment (jovial) [reply etc/kayobe/kolla/globals.yml]: Cheers, I've add the default value.

Comment (jovial) [reply etc/kayobe/kolla/globals.yml]: > If only we had merged #663 already we wouldn't need to worry about this :D

Should we try and get that merged instead?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**